### PR TITLE
Chore/add test data to agg mds test

### PR DESCRIPTION
**Pull request by haraprasadj**
Jira Ticket: [HP-942](https://ctds-planx.atlassian.net/browse/HP-942)

### CODE REVIEW

File: suites/apis/aggMDSTest.js [hunk 1]
```diff
@@ -3,80 +3,61 @@ Feature('Aggregate Metadata Service @aggMDS @requires-metadata');
...
+    users.mainAcct.accessTokenHeader, I.cache.studyId, studyMetadata,
   );
   output.print('Re-sync Aggregate Metadata');
   await mds.do.reSyncAggregateMetadata();
   const record = await mds.do.readAggMetadataRecord(
     users.mainAcct.accessTokenHeader, I.cache.studyId,
   );
   expect(record.commons_name).to.equal('HEAL');
-  expect(record.project_title).to.equal('[AUTOTEST Title] Testing Aggregate Metadata Service');
+  expect(record.project_title).to.equal(studyMetadata.gen3_discovery.project_title);
```
Comment (mfshao) [inline suites/apis/aggMDSTest.js:47]:
```suggestion
  expect(record.project_title).to.equal(projectTitle);
```

**Review by mfshao — COMMENTED** [review]
lgtm, 2 minor questions

File: .vscode/launch.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
                 "run",
                 "--verbose",
                 "--grep",
-                "@gen3ff"
+                "@wip"
```
[the remark this answers is not in the document]
Comment (haraprasadj) [reply .vscode/launch.json:22]: yes, it is just a template that makes local testing easy

**Review by mfshao — APPROVED** [review]
lgtm

**Pull request merged**